### PR TITLE
It would be grateful to contribute in this. I have modified README.rst. I am still thinking to make it more informative.

### PR DESCRIPTION
**Pull request by Nomanpathan**
Fixes #

- [ ] Add documentation.
- [ ] Add tests.
- [ ] Add a changelog entry.
- [ ] Add your name in the contributors file.
- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)
- [ ] If you added a new configuration setting, update the [`kinto.tpl`](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl) file with it.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -25,9 +25,17 @@ Kinto
 .. |pypi| image:: https://img.shields.io/pypi/v/kinto.svg
     :target: https://pypi.python.org/pypi/kinto
 
+What JSON is?
+================
+
+JSON is a popular and easy to use markup language.
+
+What Kinto is capable of?
+=========================
```
Comment (leplatrem) [inline README.rst:34]: Why not add this at the end of the introduction? 

I don't think `What JSON is ?` should be the introductory phrase

**Review by leplatrem — CHANGES_REQUESTED** [review]

**Comment by Nomanpathan** [comment]
As a advanced beginner I thought that.
When I was beginniner I was finding making parser I guess so. And I was
confused I don't know why I was confused.
But I will surely look onto whole repository, as soon as I get free from my
current stuff. I think I should and this too I don't know why

**Comment by dstaley** [comment]
@leplatrem This is likely just a spam PR as a result of [Hacktoberfest](https://blog.domenic.me/hacktoberfest/). I suggest we mark as spam, close, and lock the comments.

**Comment by leplatrem** [comment]
Oh, I had no idea! Thanks Dylan!

**Pull request closed without merging**